python/paddle/fluid/contrib/slim/tests/CMakeLists.txt

```diff
@@ -100,6 +100,14 @@ function(save_qat_model_test target qat_model_dir fp32_model_save_path int8_mode
 		 --quantized_ops ${quantized_ops})
 endfunction()
 
+function(convert_model_test target model_path model_save_path file_name pdf)
+    py_test(${target} SRCS ${CMAKE_CURRENT_SOURCE_DIR}/convert_model.py
+            ARGS --model_path ${model_path}
+	         --model_save_path ${model_save_path}
+	         --file_name ${file_name}
```

Comment:
IMO the `model_save_path` and `file_name` do not convey the purpose of the options. Maybe `save_graph_dir` and `save_graph_name`?